let a;
try {
    console.log(a.length)
} catch (error) {
    console.log(error)
}

console.log("hi")
